automation/scripts/test_migration_mysql.sh

```diff
@@ -16,24 +16,47 @@
 
 set -e
 
-if [ -z "$MLRUN_MIGRATION_MESSAGE" ]; then
-	echo "Environment variable MLRUN_MIGRATION_MESSAGE not set"
-	exit 1
-fi
-
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-export ROOT_DIR="${SCRIPT_DIR}/../.."
-
 function cleanup {
-	rm -f "${ROOT_DIR}/server/api/migrations_sqlite/mlrun.db"
+	docker kill migration-db
 }
 trap cleanup SIGHUP SIGINT SIGTERM EXIT
 
-export MLRUN_HTTPDB__DSN="sqlite:///${ROOT_DIR}/server/api/migrations_sqlite/mlrun.db?check_same_thread=false"
 
-cleanup
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+ROOT_DIR="${SCRIPT_DIR}/../.."
+
+export MLRUN_HTTPDB__DSN="mysql+pymysql://root:pass@localhost:3306/mlrun"
+
+docker run \
+	--name=migration-db \
+	--rm \
+	-v "${ROOT_DIR}:/mlrun" \
+	-p 3306:3306 \
+	-e MYSQL_ROOT_PASSWORD="pass" \
+	-e MYSQL_ROOT_HOST="%" \
+	-e MYSQL_DATABASE="mlrun" \
+	-d \
+	mysql/mysql-server:8.0 \
+	--character-set-server=utf8 \
+	--collation-server=utf8_bin
+
+times=0
+while ! docker exec migration-db mysql --user=root --password=pass -e "status" > /dev/null 2>&1; do
+	echo "Waiting for database connection..."
+	sleep 2
+	if [ $times -ge 60 ]; then
+		exit 1
+	fi
+	times=$(( times + 1 ))
+done
 
 export PYTHONPATH=$ROOT_DIR
 
-alembic -c "${ROOT_DIR}/server/api/alembic.ini" upgrade head
-alembic -c "${ROOT_DIR}/server/api/alembic.ini" revision --autogenerate -m "${MLRUN_MIGRATION_MESSAGE}"
+cd ${ROOT_DIR}/server/api
```

Comment:
same about cd, think you may be able to cd without changing the cwd like in `make compile-schemas`